config/locales/fr.yml

```diff
@@ -177,29 +177,29 @@ fr:
         text: Besoin d'un nouveau mot de passe ? Pas de problème. %{link}
         title: Renvoi de mot de passe
       delete:
-        delete:
-        delete_profile:
-        warning:
+        delete: Supprimer
+        delete_profile: Supprimer le profil
+        warning: La suppression de votre profil est une action irréversible. Cela ne peut pas être annulé en contactant le support. Soyez sûr !
```

Comment:
"Soyez sur". Maybe "Soyez en sur"? 🤔

Reply:
The [original](https://github.com/rubygems/rubygems.org/blob/master/config/locales/en.yml#L182) is "Please be certain!". Indeed "Soyez sûr" isn't exactly fluid. I'd maybe suggest something like "Réfléchissez-bien avant de continuer !" or another type of sentence!